portability-api/src/main/java/org/datatransferproject/api/action/transfer/GetTransferJobAction.java

```diff
@@ -34,6 +34,7 @@ public TransferJob handle(GetTransferJob transferRequest) {
 
     PortabilityJob job = jobStore.findJob(jobId);
 
-    return new TransferJob(id, job.exportService(), job.importService(), job.transferDataType(), null, null);
+    return new TransferJob(id, job.exportService(), job.importService(), job.transferDataType(),
+            null, null, null, null, null, null);
```

Comment:
This looks like we should create a static factory method in TransferJob for initiating clients with this state.

Reply:
I left a TODO in TransferJob that should take care of this:

TODO(#553): Refactor String params to make less error-prone.

I'll add a comment here too